### DISCUSSION TITLE
Short URL already exists in database or is reserved

### DISCUSSION THREAD

**Discussion by drwMrk** [YOURLS/YOURLS#3613] [Q&A] [answered]
I'm attempting to redirect 1stlu.org/blood to go to https://donate.wisconsin.versiti.org/donor/schedules/drive_schedule/258101.

However, when I try to create the short URL, YOURLS presents this error "Short URL blood already exists in database or is reserved"

I think this is correct, however I'm unable to find 1stlu.org/blood in the database. I can't manually find it, no matter how I sort the database (my database currently has only 45 entries, so it is easy enough to search manually). And when I search the database, it does not come up in the search.

I do have 1stlu.org/blooddrive ALSO pointing to https://donate.wisconsin.versiti.org/donor/schedules/drive_schedule/258101... but I have YOURLS setup to allow duplicate short URLS, so this shouldn't be an issue.

Any thoughts as to why I can't see the short URL that I need to edit would be appreciated.

**Comment by drwMrk** [comment, marked as answer]
So it appears I may have answered my own question, though I'm not certain why this would be. But instead of using Chrome to manage this short URL (and not being able to find it), I opened my YOURLS page in Safari, and both 1stlu.org/blood and 1stlu.org/blooddrive show in YOURLS when I'm using Safari.

**Reply by dgw** [reply]
The most likely cause is a misbehaving (or unexpectedly behaving) Chrome extension, if using Safari instead fixes the visibility issue. If you have anything like uBlock Origin installed, that (class of) extension(s) would be my first suspect.
